### ISSUE TITLE
Bug: it writes text without dot and comma

### ISSUE THREAD

**Issue by davidak** [coqui-ai/STT#1873]
**Describe the bug**
it writes text without dot and comma

**To Reproduce**

```
tts --text "To help with the large amounts of pull requests, we would appreciate your reviews of other pull requests, especially simple package updates. Just leave a comment describing what you have tested in the relevant package/service. Reviewing helps to reduce the average time-to-merge for everyone. Thanks a lot if you do!"

[nix-shell:~]$ stt --model coqui-stt-0.9.3-models.pbmm --scorer coqui-stt-0.9.3-models.scorer --audio '/home/davidak/Downloads/tts-0.0.14.wav' 
TensorFlow: v2.3.0-6-g23ad988fcde
 Coqui STT: v0.10.0-alpha.4-74-g49cdf7a6
2021-05-21 23:11:10.956472: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
rate: rate clipped 1 samples; decrease volume?
to help with the large amounts of pull requests we would appreciate your reviews of other poll requests especially simple package up dates just leave a comment describing what you have tested in the relevant packages service reviewing helps to reduce the average time to merge for every one thanks a lot if you do
```

**Expected behavior**
In the best case, it would be exactly the same as the input here.

I also tried with spoken words from me and it ignored pauses and intonation.

**Environment (please complete the following information):**
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: NixOS 21.05pre289526.7a1fbc38a4b
- **TensorFlow installed from (our builds, or upstream TensorFlow)**:
- **TensorFlow version (use command below)**: v2.3.0-6-g23ad988fcde
- **Python version**:
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

**Comment by reuben** [comment]
That's not a bug, we normalize the training data to remove punctuation. It may be a feature request :)

**Comment by kdavis-coqui** [comment]
@davidak It's possible to train a model in which the training data is normalized to include punctuation. Such a model will include punctuation in its transcription.

So if you feel up to it you could train your own model to transcribe with punctuation!

**Comment by davidak** [comment]
in that case take it as a feature request

Am 22. Mai 2021 09:07:54 MESZ schrieb Reuben Morais ***@***.***>:
>That's not a bug, we normalize the training data to remove punctuation.
>It may be a feature request :)
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/coqui-ai/STT/issues/1873#issuecomment-846366343

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.
